docs/modules/operation/pages/events/event-definition.adoc

```diff
@@ -0,0 +1,57 @@
+
+[[ga-events-event-definition]]
+= Event Definition
+
+
+== Associate an event to a specific node
+
+There are two ways to associate an existing node to a given event prior sending it to the event bus:
+
+* Set the `nodeId` field of event to the database ID of the desired node.
+* For requisitioned nodes, set the `_foreignSource` and `_foreignId` as parameters to the event.
+Then, any incoming event without a `nodeId` and these two parameters will trigger a lookup on the database.
+If a node is found, the `nodeId` attribute will be dynamically set on the event, regardless of which method has been used to send it to the event bus.
+
+
+[[ga-events-anatomy-of-an-event]]
+== Anatomy of an event
+
+Events are structured historical records of things that happen in {page-component-title} and the nodes, interfaces, and services it monitors.
+Every event has a number of fixed fields and zero or more parameters.
+
+.Mandatory Fields
+UEI (Unique Event Identifier)::
+    A string that uniquely identifies the event's type.
+    UEIs are typically formatted in the style of a URI, but the only requirement is that they start with the string `uei.`.
+Event Label::
+    A short, static label that summarizes the gist of all instances of this event.
+Description::
+    A long-form description of all instances of this event.
+    HTML formatting can be used to format the body of the event text.
+Log Message::
+    A long-form log message describing this event, optionally including expansions of fields and parameters so that the value is tailored to the event at hand.
+    The possible options for this setting are:
+    logndisplay:::
+        Both log the event in the database and display it in the Web UI.
+    logonly:::
+        Log the event in the database, but do not display it.
+    suppress:::
+        Neither log the event in the database or display it.
+    donotpersist:::
+        Do not log the event in the database, but still send it to daemons that are listening for this type of event, such as being used to generate notifications.
+        Note that eventd and alarmd don't process this type of events.
```

Comment:
Let me know either way and I can find-replace.  I am once again deferring to your guidance on this point.

Reply:
I think we should stick with capitalizing daemon names, just to make sure there's no confusion.

Reply:
Did a bit of research and confirmed that it's standard to spell daemons lower case, since Unix doesn't like caps. I recommend we use lowercase for all daemons (except at the start of sentences). We can update as we go through the docs. We could put them in `` marks to offset them, but RedHat doesn't do this with systemd, for example, so it would be best to minimize the extra formatting. @emarsh1995 could you add this to the style guide?

Reply:
Will do!